### DISCUSSION TITLE
Custom layout.antlers.html for one template

### DISCUSSION THREAD

**Discussion by jannejava** [statamic/cms#5970] [Q&A] [answered]
Hi!

I am aware you can set a template directly inside a page's md-file like`layout: layout_inverted`

However, is it possible to set a corresponding layout file to a template automatically? Every time a page with template X is created it should use template layout_inverted by default, without specifying it in the page's markdown file.

Or am I doing something wrong and you should never have more than one layout-file? 

Thanks in advance!

/ Jan

**Comment by joshuablum** [comment, marked as answer]
Hi Jan,

Let me think 🤔

I know that you can do something like `template: '@blueprint'` ([link to docs](https://statamic.dev/views#inferring-templates-from-entry-blueprints)). Maybe do some kind of conditional in the blueprint then? More on that [here](https://statamic.dev/blueprints#conditional-fields).

> Or am I doing something wrong and you should never have more than one layout-file?

It's perfectly fine to have more than one layout file. It's even mentioned in the docs [here](https://statamic.dev/views#layouts).

Hope that helps!

**Reply by jannejava** [reply]
Thanks for the reply! I will check it out!
